packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -152,6 +163,12 @@ export async function createLanguageServer(
     }
   };
 
+  if (enableApexRename) {
```

Comment:
Had to add it this way or else the `Rename Symbol` option will always show in the menu regardless of the status of the feature flag.